src/Symfony/Bridge/Twig/NodeVisitor/TranslationNodeVisitor.php

```diff
@@ -158,6 +158,19 @@ private function getReadDomainFromNode(Node $node): ?string
             return $node->getAttribute('value');
         }
 
+        if (
+            $node instanceof FunctionExpression
```

Comment:
Based on the updated fixture I would have expected a `FilterExpression` here. But that makes me think that we probably need to provide the fix for the `trans` tag, `t()` function as well as the `trans` filter.

Reply:
I added test and they are now supported https://github.com/symfony/symfony/pull/53357/commits/52b111aab733eccda7799f0449d247be9fc2a148